manifests/provisioners/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+bases:
+- ./plain
+
```

Comment:
Note: we could also add a kustomize-related Makefile target for installing the rukpak provisioners, e.g. `make install-provisioners`, given we already have a dedicated plain provisioner Makefile target. I know @joelanford is a big fan of introducing new targets too.

Reply:
Lol.  I'm a +1 to this PR if we remove the existing `install-plain-provisioner` and replace it with `install-provisioners`

Reply:
Done - just ran the `install-provisioners` target locally and ran into no errors. I didn't see anything in our documentation that indicates we use any Makefile target outside of `make run` for quickstart purposes.